tools/accuracy_checker/accuracy_checker/adapters/face_recognition_quality_assessment.py

```diff
@@ -0,0 +1,79 @@
+"""
+Copyright (c) 2020 Intel Corporation
```

Comment:
please copyrigth align for new files
```suggestion
Copyright (c) 2018-2020 Intel Corporation
```